llvm/utils/mlgo-utils/mlgo/corpus/extract_ir.py

```diff
@@ -156,10 +88,86 @@ def main(argv):
     )
 
 
-def entrypoint():
-    multiprocessing.set_start_method("fork")
-    app.run(main)
-
-
 if __name__ == "__main__":
-    entrypoint()
+    parser = argparse.ArgumentParser(
```

Comment:
nit: could you factor all this in a function and put it at the top - this would be consistent to where folks tend to look for module-defined flags.

Reply:
Good point. Moved them into a new `parse_args_and_run` function that is at the top of the file.